pynamodb/models.py

```diff
@@ -13,7 +12,7 @@
 from pynamodb.expressions.update import Action
 from pynamodb.exceptions import DoesNotExist, TableDoesNotExist, TableError, InvalidStateError, PutError
 from pynamodb.attributes import (
-    Attribute, AttributeContainer, AttributeContainerMeta, MapAttribute, TTLAttribute, VersionAttribute
```

Comment:
interesting to see this was unused - i wonder if we should include a pre-commit linter or something going forward